src/System.Buffers.Experimental/System/Buffers/IReadOnlyBufferList.cs

```diff
@@ -5,12 +5,15 @@
 
 namespace System.Buffers
 {
-    public interface IReadOnlyBufferList<T> : ISequence<ReadOnlyMemory<T>>
+    public interface IReadOnlyMemorySequence<T> : ISequence<ReadOnlyMemory<T>>
     {
         int CopyTo(Span<T> buffer);
```

Comment:
Is `CopyTo` required? Why can't it be implemented using `TryGet`?

Reply:
This is both very convenient (for dealing with straddling segments) and faster than using TryGet.

Reply:
Why is it faster? First segment can be read from `Memory` and `TryGet` for other segments.

Reply:
Requires 3 calls and lots of passing of largish structs around.